.travis.yml

```diff
@@ -1,5 +1,25 @@
 language: ruby
-rvm:
-  - 2.2.4
-before_script:
+dist: xenial
+os: linux
+
+branches:
+  only:
+  - master
+  - develop
+
+addons:
+  postgresql: '9.6' # Match ricksteves-prod
```

Comment:
fwiw, it looks like ricksteves-prod is on 11.20 (https://data.heroku.com/datastores/49e0fe33-2c61-4762-98d9-2b0f924ed528)

Reply:
This will get changed in the next PR.